### PR TITLE
Misc changes

### PR DESCRIPTION
**Pull request by W95Psp**
cc @karthikbhargavan: this PR addresses some of what we discussed this morning, wdyt?

### CODE REVIEW

File: proof-libs/fstar/core/README.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+The second stage of our approach to `core` is automatic generation
+with specifications and models.
+Our plan is to annotate the Rust `core` library with specifications
+and models written directly as Rust annotations.
+This will enable automatic generation of `core` models with consistent
+semantics in all of hax backends (for now F\* and Coq).
+
+Note that we already started experimenting with this second approach:
+hax is already able to digest and generate signature-only F\* for a
+huge part of core.
```
Comment (karthikbhargavan) [inline proof-libs/fstar/core/README.md:28]: instead of "huge" let's say 30% or 70%?

File: examples/README.md [hunk 2]
```diff
@@ -3,16 +3,16 @@
 | Name     | Status of the F* extraction |
 | -------- | -------------- |
```
Comment (karthikbhargavan) [inline examples/README.md:4]: Lax-typechecks -> Typechecks?

File: examples/README.md [hunk 3]
```diff
@@ -3,16 +3,16 @@
...
-Thus, to generate the F* modules of an example `<EXAMPLE>` and to
-**typecheck** them, you should:
-1. move to the proper `extraction` subdirectory: `cd <EXAMPLE>/proofs/fstar/extraction`;
-2. run `make`: `make` (or `OTHERFLAGS="--lax" make` if the example is only lax-checking).
+Running `make` will run `make` in each example directory, which in
+turn will generate F\* modules using hax and then typecheck those
+modules using F\*.
 
+Note the generated modules live in the
+`<EXAMPLE>/proofs/fstar/extraction` folders.
```
Comment (karthikbhargavan) [inline examples/README.md:18]: Say something at the end for Coq code generation?

**Review by karthikbhargavan — APPROVED** [review]
Except for the small nits, looks good.

File: proof-libs/fstar/core/README.md (hunk 1 above)
Comment (W95Psp) [reply proof-libs/fstar/core/README.md]: according to my numbers, hax extracts 19k definitions over 20k, so it's more 95% :smiley:
Gonna write "more than 80%", some are extracting very badly

**Pull request merged**